packages/shared/src/helpers/networks.ts

```diff
@@ -13,7 +13,8 @@ export enum Networks {
   Polygon = "polygon",
   Moonbeam = "moonbeam",
   Pendulum = "pendulum",
-  Stellar = "stellar"
+  Stellar = "stellar",
+  PolygonAmoy = "polygonAmoy"
```

Comment:
Maybe we should add this type to the enum only conditionally based on the sandbox config so that it's not automatically shown on the network selector